ada_2_fhir/zibs2017/payload/zib-procedure-2.1.xsl

```diff
@@ -261,11 +261,13 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                         </xsl:choose>
                     </status>
                     
-                    <!-- category is required in the FHIR profile, so always output category, data-absent-reason if no actual value -->
+                    <!-- category is required in the FHIR profile, and 387713003 is needed for BgZ MM -->
                     <category>
-                        <extension url="{$urlExtHL7DataAbsentReason}">
-                            <valueCode value="unknown"/>
-                        </extension>
+                        <coding>
+                            <system value="http://snomed.info/sct"/>
+                            <code value="387713003"/>
+                            <display value="chirurgische ingreep"/>
+                        </coding>
                     </category>
```

Comment:
We could do something like 'if $usecase = 'bgz-mm' then ... else ... here - but that would be a first I guess

Reply:
That seems like a nice solution. In the BgZ MSZ qualification material there is a procedure 'aanleggen van loopgips voor onderbeen' and that doesn't sound like a 'chirurgische ingreep' at all ;)